frontend/packages/ceph-storage-plugin/src/components/namespace-store/namespace-store-dropdown.tsx

```diff
@@ -27,15 +27,24 @@ export const NamespaceStoreDropdown: React.FC<NamespaceStoreDropdownProps> = ({
 }) => {
   const { t } = useTranslation();
   const [isOpen, setOpen] = React.useState(false);
-  const [nnsObj, nnsLoaded, nnsErr] = useK8sGet<ListKind<NamespaceStoreKind>>(
-    NooBaaNamespaceStoreModel,
-    null,
+  const [dropdownItems, setDropdownItems] = React.useState<JSX.Element[]>([]);
+
+  const nooBaaNsResource = {
+    kind: referenceForModel(NooBaaNamespaceStoreModel),
+    name: null,
+    isList: true,
```

Comment:
move it to resource.ts
```suggestion  
  const namespaceStoreResource = {
    kind: referenceForModel(NooBaaNamespaceStoreModel),
    name: null,
    isList: true,
```